.github/workflows/integration-opendistro

```diff
@@ -0,0 +1,18 @@
+name: Integration tests with OpenDistro
```

Comment:
file extension is missing. ".yml" . I believe thats why this job is not picked up by github workflow.

Reply:
Oops!! I fixed it